src/components/CharacterInfo.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import styles from "styled-components";
+
+const CharInfoStyles = styles.div`
+div{
+    border:5px dashed #a9def9;
+    width:10%
+}
+
+
+    color: black;
+    
+
+
+
```

Comment:
Try to eliminate this whitespace. Strive to leave few whitespace as a rule of thumb